gini/gini-vision-lib-ios
# Add settings to example app

Added settings menu in the select api screen, so it is possible to test different configuration without having to run and build the app again.

### How to test
Press in the metadata text in the select api screen and see that a new screen appears. Try to change one of the settings and then see that it is applied both in the Screen API and Component API

### Merging
Automatic.

### Ticket 
Issue #109
